.github/workflows/do_not_merge.yaml

```diff
@@ -0,0 +1,41 @@
+name: "Check commits can be merged"
```

Comment:
@andy-slac I think I have something working now. Can you take a quick look at this action and review it please?